### PR TITLE
Closes #88 add visualization to dashboard

### LINKED ISSUE #88: dashboard - add visualization
add a saved visualization to an existing dashboard

### PR DESCRIPTION
**Pull request by jpkeyw**
Both the dashboard and visualization to add to it need to already exist.

### CODE REVIEW

**Comment by jeffshaver** [comment]
on the create form, tabbing through goes from title to subtitle to submit instead of title to subtitle to group to submit

**Comment by pml984** [comment]
Appears to be an issue with the DropdownMenu in material

https://github.com/callemall/material-ui/pull/4966

**Comment by jeffshaver** [comment]
when you click add visualization, the popover should appear from the top right and end at the top left. the current animation (top left to bottom right) actually threw me off when seeing it

**Comment by jeffshaver** [comment]
react-loader looks nothing like the current loader we use (from material-ui/safe-framework). seems out of place

**Comment by jeffshaver** [comment]
after adding a visualization to a blank dashboard, the visualization toolbar is getting hidden

**Comment by jeffshaver** [comment]
after adding more than one visualization to a blank dashboard, the grid is completely messed up. things overlap, can't drag to increase size correctly, etc...

**Comment by jeffshaver** [comment]
after clicking save (after adding visualizations), all my visualizations go away until i refresh

File: package.json [hunk 1]
```diff
@@ -75,6 +77,7 @@
     "react-dom": "^15.3.1",
     "react-grid-layout": "^0.13.1",
     "react-leaflet": "^0.12.1",
+    "react-loader": "^2.4.0",
```
Comment (jeffshaver) [inline package.json]: this doesn't seem to be used anywhere

**Pull request merged**